test/e2e/virtual/apiexport/virtualworkspace_test.go

```diff
@@ -413,6 +456,57 @@ func TestAPIExportPermissionClaims(t *testing.T) {
 		[]string{ul.Items[0].GetName(), ul.Items[1].GetName()},
 		[]string{"in-vw", "in-vw-before"},
 	))
+
+	var newClaims []apisv1alpha1.PermissionClaim
+	for i := range apiExport.Spec.PermissionClaims {
+		claim := apiExport.Spec.PermissionClaims[i]
+		if claim.Group == "" && claim.Resource == "configmaps" {
+			continue
+		}
+		newClaims = append(newClaims, claim)
+	}
+
+	updatedExport := apiExport.DeepCopy()
+	updatedExport.Spec.PermissionClaims = newClaims
+
+	oldJSON = encodeJSON(t, apiExport)
+	newJSON = encodeJSON(t, updatedExport)
+	patchBytes, err = jsonpatch.CreateMergePatch(oldJSON, newJSON)
+	require.NoError(t, err, "error creating patch")
+	t.Logf("patching apiexport to remove claim on configmaps")
+	_, err = kcpClusterClient.ApisV1alpha1().APIExports().Patch(logicalcluster.WithCluster(ctx, serviceProviderWorkspace), apiExport.Name, types.MergePatchType, patchBytes, metav1.PatchOptions{})
+	require.NoError(t, err, "error patching apiexport")
```

Comment:
same as above, please elaborate on Patch vs Update

Reply:
Will switch to update

Reply:
Actually, going to keep patch to avoid conflicts